pages/index.tsx

```diff
@@ -70,14 +70,10 @@ const Home: NextPage<HomePageProps> = ({ searchIndex }) => {
       <main className="m-4 l:m-0">
         <div className="max-w-4xl w-4xl mx-auto mt-8 flex flex-col items-center">
           <h1 className="text-bzl-green font-bold text-6xl">
-            Bazel Central Registry
+            Bazel Module Registry
```

Comment:
revert this?

Reply:
That was intentional as part of rewording the H2 content below it to be less informal. The "central registry" is the other repository. But sure, I'm not that opinionated about it.